cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-renewed.cy.js

```diff
@@ -0,0 +1,253 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import TestTypes from '../../../../support/dictionary/testTypes';
+import devTeams from '../../../../support/dictionary/devTeams';
+import permissions from '../../../../support/dictionary/permissions';
+import UserEdit from '../../../../support/fragments/users/userEdit';
+import TopMenu from '../../../../support/fragments/topMenu';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+import generateItemBarcode from '../../../../support/utils/generateItemBarcode';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../../../support/fragments/settings/users/patronGroups';
+import Location from '../../../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../../../support/fragments/users/users';
+import CirculationRules from '../../../../support/fragments/circulation/circulation-rules';
+import CheckInActions from '../../../../support/fragments/check-in-actions/checkInActions';
+import Checkout from '../../../../support/fragments/checkout/checkout';
+import ServicePoints from '../../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import UsersOwners from '../../../../support/fragments/settings/users/usersOwners';
+import PaymentMethods from '../../../../support/fragments/settings/users/paymentMethods';
+import LoanPolicy from '../../../../support/fragments/circulation/loan-policy';
+import Conditions from '../../../../support/fragments/settings/users/conditions';
+import Limits from '../../../../support/fragments/settings/users/limits';
+import UsersSearchPane from '../../../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../../../support/fragments/users/usersCard';
+import UserLoans from '../../../../support/fragments/users/loans/userLoans';
+
+function generateUniqueItemBarcodeWithShift(index = 0) {
```

Comment:
may be better to move this function into support/utils
It seems like reusable and valuebale function

Reply:
done